source/base/parameter_acceptor.cc

```diff
@@ -59,12 +59,7 @@ ParameterAcceptor::ParameterAcceptor(const std::string &name)
 ParameterAcceptor::~ParameterAcceptor()
 {
   std::lock_guard<std::mutex> l(class_list_mutex);
-  // Notice that it is possible that the class is no longer in the static list.
-  // This happens when the clear() method has been called. We must guard that
-  // this is not the case, and therefore we only remove ourselves from the list
-  // if we are actually there.
-  if (class_list.find(this) != class_list.end())
-    class_list.erase(this);
+  class_list.erase(this);
```

Comment:
I think the comment was useful, it's just that `erase` is documented as doing nothing if the key was not found. Would you mind updating, rather than removing, the comment?